### ISSUE TITLE
License Violation

### ISSUE THREAD

**Issue by Minesraft2** [MrPelkie/Blooket-Hacks-For-SMS#1]
The scripts in this repository are clearly stolen from mine. I have not given you permission to repost these cheats, plus you removed the license part of each one. Please private or remove this repo within 2 hours or I will send a DMCA takedown request.

**Comment by MrPelkie** [comment]
We have got these scripts for multiple webs, we have also got permission for numerous creators to use this script. Also you have not had permissions from our members and group to copy there scripts from our numerous projects. Do not copy our scripts or we will have a DMCA request sent, goodbye.
-Team Apollo.

> The scripts in this repository are clearly stolen from mine. I have not given you permission to repost these cheats, plus you removed the license part of each one. Please private or remove this repo within 2 hours or I will send a DMCA takedown request.

**Comment by Minesraft2** [comment]
Out of the 12 scripts in this repository, 11 were created by me. From scratch. No more 2 hours, I'll submit the dmca now.

**Comment by MrPelkie** [comment]
> Out of the 12 scripts in this repository, 11 were created by me. From scratch. No more 2 hours, I'll submit the dmca now.

Hello, I am the leader of this group, I am so sorry the my team has not reached out to me about this.
I am trying to resolve this issue, we will give credits for these scripts in our README.md. I taught all of my members how to make ZEN scripts, Blooket scripts, Roblox, etc and they still took them, Those who took them without perms are facing punishment. - Perxu  - Co-founder of Team Apollo

**Comment by Minesraft2** [comment]
Credited or not, reposting my scripts goes against the attached license. I will not be retracting the dmca.

**Comment by MrPelkie** [comment]
> Credited or not, reposting my scripts goes against the attached license. I will not be retracting the dmca

**Comment by MrPelkie** [comment]
> Credited or not, reposting my scripts goes against the attached license. I will not be retracting the dmca.

I am looking in the scripts and I do not see a license, are you positive it is yours?

**Comment by Minesraft2** [comment]
Compare [my daily tokens script](https://github.com/Minesraft2/Blooket-Cheats/blob/main/unobfuscated/global/getDailyRewards.js) with [yours](https://github.com/MrPelkie/Blooket-Hacks-For-SMS/blob/main/Tokens)

**Comment by MrPelkie** [comment]
Can you provide the license to review?

**Comment by Minesraft2** [comment]
```js
/**
* @license StewartPrivateLicense-2.0.0
* Copyright (c) Aerell McKnight 2023
*
* You may not reproduce or distribute any code inside this file without the licenser's permission.
* You may not copy, modify, steal, skid, or recreate any of the code inside this file.
* You may not under any circumstance republish any code from this file as your own.
* 
* ALL TERMS STATED IN THE LINK BELOW APPLY ASWELL
* https://github.com/Minesraft2/Blooket-Cheats/blob/main/LICENSE
*/
```
https://github.com/Minesraft2/Blooket-Cheats/blob/main/LICENSE

**Comment by MrPelkie** [comment]
"@license StewartPrivateLicense-2.0.0
> * Copyright (c) Aerell McKnight 2023" Is not a vailed/official license in the database, therefore actions cannot be took 
-perxu

**Comment by Minesraft2** [comment]
That's not how Github does dmcas. There are terms stated in the repo and if they are violated Github can take the repo down.

**Comment by Minesraft2** [comment]
It does not matter if you're going to continue trying to avoid the dmca, it will not work. I will not retract it. This issue is resolved.

**Comment by MrPelkie** [comment]
> It does not matter if you're going to continue trying to avoid the dmca, it will not work. I will not retract it. This issue is resolved.

This is a team corporation and issues will be resulted once vailed license is provided

**Comment by MrPelkie** [comment]
what???

**Comment by Minesraft2** [comment]
Let me say it in simpler terms for you, "take repo down or github make it go bye bye"

**Comment by MrPelkie** [comment]
> Let me say it in simpler terms for you, "take repo down or github make it go bye bye"

The entire project will not be took down. I will contact github to resole it, as of right now i will not take action. bye bye have a nice day

**Comment by Minesraft2** [comment]
No matter what you do, my scripts will not stay on this project. So yes, I will have a nice day.

**Comment by MrPelkie** [comment]
if you can provide proof of creating the scripts we cannot take actions, once proof is provided it will be took down. Kiptyn- Ceo and founder

**Comment by GooseterV** [comment]
The license is 100% valid and github WILL take action on your repository regardless of whether you continue this nonsense or if you quietly resign. As the creator and copyright holder of the original license itself, the license has full validation from multiple trusted big corporations such as:
- Google (YouTube)
- GitHub
etc.
Any further you continue to argue will not have any new outcome on what has already been done. If you would like to apologize, "all" of you will be forgiven.
Any deviations from the rules of the Stewart Private License violate copyright laws and the DMCA, you are blabbering about "invalid licenses".

**Comment by MrPelkie** [comment]
From Apollo, we will apologize for the actions of our members. But in our teams TOS we can't take actions without any proof provided. Our project will remain up as of now, but we will credit your team. - sincerely kiptyn

**Comment by GooseterV** [comment]
We have provided you with all the proof necessary. We do not have a "team" as we are individuals on our own accounts. You and your circus are unable to comprehend that licensing is valid and seeing as you do not want to accept that you are in the wrong, the choice has already been made to dmca this repository, and has been submitted.

**Comment by GooseterV** [comment]
If you do not see my license as valid, gladly take it up with github themselves. I would be overjoyed for you to explain to github why my license is invalid. Unfortunately you do not either have the balls nor the authority to do this, as you are a newly created account which an 11 year old boy has infested with a so-called "team" of made up people and names.

**Comment by GooseterV** [comment]
I would also like to add that if you consider yourself a "team", we will not validly acknowledge this until the individuals have separate accounts and can speak for themselves. You have a single repository which is filled with programming malpractices, something that only a toddler could somehow create. Any further discussion from you and your so-called "team" is pointless as the past cannot be undone.

**Comment by MrPelkie** [comment]
rxzyx was a previous account of our group that has got taken down. Also you have harassed my team multiple times and we will be taking action

**Comment by GooseterV** [comment]
Go ahead, get your circus to take action on me. I dare you. You know why? Because i know you children cannot do a single thing. You are clearly in the legal and moral wrong, violating the DMCA's rules and copyright laws. If you are going to go down this path, you better know what youre getting yourself into.

**Comment by GooseterV** [comment]
The license which i have created has the following terms inside of it (i will be providing simplified terms).
- you may not use the code in any other way other than personal or social USE.
- you cannot advertise or lock behind purchasing.
- you cannot reupload the code in any way
- you cannot in any way modify and redistribute said code.
- you may not claim said code is your own.

**Comment by MrPelkie** [comment]
![90f3c2b9fca4f1efa215f680ba2114e1](https://user-images.githubusercontent.com/123598636/214970193-a6181359-6e40-4cad-8be7-f0bbf8700233.jpg)

**Comment by GooseterV** [comment]
Along with your "group", rx. was differentiating from the terms and conditions listed in my license, and was rightfully struck down, as he was breaking the DMCA's rules. If you feel the need to continue this discussion, make sure you and your "team" knows that i am not ready to backdown anytime soon.

**Comment by Minesraft2** [comment]
> ![90f3c2b9fca4f1efa215f680ba2114e1](https://user-images.githubusercontent.com/123598636/214970193-a6181359-6e40-4cad-8be7-f0bbf8700233.jpg)

💀

**Comment by MrPelkie** [comment]
> > ![90f3c2b9fca4f1efa215f680ba2114e1](https://user-images.githubusercontent.com/123598636/214970193-a6181359-6e40-4cad-8be7-f0bbf8700233.jpg)
> 
> 💀

**Comment by GooseterV** [comment]
Have you given up @MrPelkie ?
If you wish to concede and rest your directed accusations and claims of falsity of my content i will be more than happy to escort you out the door, although right now you don't seem to have anything to say to any of this.

**Comment by Minesraft2** [comment]
Ok well, we have actual lives to get to. Enjoy the rest of your time in middle school. See you after the dmca.

**Comment by GooseterV** [comment]
See above.

**Comment by MrPelkie** [comment]
> Ok well, we have actual lives to get to. Enjoy the rest of your time in middle school. See you after the dmca.

First, I am taking scripting classes in university of northwestern

**Comment by GooseterV** [comment]
Oh really? And what nonsense are you going to come up with this time? "I invented progaming langage. Feer me you Stupid." or are you going to peacefully resign. I have plenty of time to shame you.

**Comment by MrPelkie** [comment]
what? im 19

**Comment by GooseterV** [comment]
Your age is irrelevant considering you have the mental age and comprehension of a young primary schooler. I will again ask you to peacefully resign your claims.

**Comment by monkxy** [comment]
you're so stupid for a 19 year old 😹 you can't even read a license and abide by the terms you're funny

**Comment by monkxy** [comment]
> > Let me say it in simpler terms for you, "take repo down or github make it go bye bye"
> 
> 
> 
> The entire project will not be took down. I will contact github to resole it, as of right now i will not take action. bye bye have a nice day

resolve* learn to fucking speed you're 19 💀

**Comment by MrPelkie** [comment]
> > > Let me say it in simpler terms for you, "take repo down or github make it go bye bye"
> > 
> > 
> > The entire project will not be took down. I will contact github to resole it, as of right now i will not take action. bye bye have a nice day
> 
> resolve* learn to fucking speed you're 19 💀

learn to speed you mean " spell "

**Comment by GooseterV** [comment]
@monkxy While your comments may be true, they are unnecessary at the present moment and i would like to ask you to tone down the blatant rudeness.

**Comment by GooseterV** [comment]
@MrPelkie Have you found another thing to stall with? Are you going to resign yet?

**Comment by GooseterV** [comment]
It'd all be for the better of us if you simply backed down and either removed the repository or accepted the takedown.

**Comment by monkxy** [comment]
> @monkxy While your comments may be true, they are unnecessary at the present moment and i would like to ask you to tone down the blatant rudeness. 

👍

**Comment by MrPelkie** [comment]
Me sped?!? Aren't you in a group made by a 9yr old titled " Council-of-Ben "?

**Comment by monkxy** [comment]
> > > > Let me say it in simpler terms for you, "take repo down or github make it go bye bye"
> 
> > > 
> 
> > > 
> 
> > > The entire project will not be took down. I will contact github to resole it, as of right now i will not take action. bye bye have a nice day
> 
> > 
> 
> > resolve* learn to fucking speed you're 19 💀
> 
> 
> 
> learn to speed you mean " spell "

meh, a quote on quote college student should know more about plagiarism that a middle schooler 🤷‍♂️

**Comment by Greenlio** [comment]
@monkxy isnt in the council of ben lol

**Comment by monkxy** [comment]
> @monkxy isnt in the council of ben lol

yeah yeah

**Comment by Greenlio** [comment]
i made the org, and for reference, it's made as a group to share code between friends - none of which are younger than 14 - and for a group owned by people significantly younger than you, the people in question are very right and you will indeed suffer the consequences of your actions and meet the same fate as your late friend, rzx

**Comment by MrPelkie** [comment]
if you look at the issues all of you submit to EVERY SINGLE BLOOKET HACK scripter it seems like all of you copy everyone

**Comment by MrPelkie** [comment]
> i made the org, and for reference, it's made as a group to share code between friends - none of which are younger than 14 - and for a group owned by people significantly younger than you, the people in question are very right and you will indeed suffer the consequences of your actions and meet the same fate as your late friend, rzx

im not scared

**Comment by GooseterV** [comment]
quite the contrary if you would comprehend this simple talk... many of the "cheats" which currently exist is a frankenrepo of our scripts and scripts which we have made and loaned to others and/or taught how to make. Jod is probably **the** main cause of blooket cheats that people frenzy over now.

**Comment by Greenlio** [comment]
> if you look at the issues all of you submit to EVERY SINGLE BLOOKET HACK scripter it seems like all of you copy everyone

no, everyone copies @Minesraft2, one of the most popular blooket cheat creators at this time

**Comment by monkxy** [comment]
> > i made the org, and for reference, it's made as a group to share code between friends - none of which are younger than 14 - and for a group owned by people significantly younger than you, the people in question are very right and you will indeed suffer the consequences of your actions and meet the same fate as your late friend, rzx
> 
> 
> 
> im not scared

the way you stoped using your wannabe business grammar indicates that you are scared

**Comment by GooseterV** [comment]
@MrPelkie Your argument about proofs and evidence that we own the scripts can be reversed right towards you. Where is the evidence that you did not just blatantly plagiarise and copy our content and code? If you are unable to provide this evidence, then we will take whatever action as we see fit, as we are in the moral and legal right here.

**Comment by MrPelkie** [comment]
listen i didn't copy from you, i copied from another project

**Comment by monkxy** [comment]
@MrPelkie and if you genuinely are in college this is dumb of you considering you should know plagiarism can get you kicked out of college.

**Comment by GooseterV** [comment]
The code is exactly the same as our licensed code in our project. Said licensing still applies.

**Comment by MrPelkie** [comment]
the license were not in the script, so you cant take action

**Comment by monkxy** [comment]
> listen i didn't copy from you, i copied from another project

that's still plagiarism, copying someone's code is plagiarism regardless unless you have permission

**Comment by GooseterV** [comment]
I can in fact take action. Your scripts are direct reproduction of the licensed code, and therefore is still breaking the terms of the license.

**Comment by MrPelkie** [comment]
well its not on there so too bad

**Comment by GooseterV** [comment]
As you can see publicly on github, for everyone to see, the scripts are licensed in their own repository, and even if you remove the license to re-upload them, the code is still stolen and is a direct reproduction so therefore is violating the terms.

**Comment by monkxy** [comment]
> the license were not in the script, so you cant take action 

yes they can 💀 the code is identical to the licensed code meaning THE LICENSE APPLIES!!!!!!!! a quote on quote college student should know more about plagiarism especially because you can get kicked out of school for that.

**Comment by monkxy** [comment]
> As you can see publicly on github, for everyone to see, the scripts are licensed in their own repository, and even if you remove the license to re-upload them, the code is still stolen and is a direct reproduction so therefore is violating the terms.

you put this way better than i could

**Comment by BenStewartYT** [comment]
> well its not on there so too bad

You sound like a 10-year-old to me. Welcome to the Blooket administration team!

**Comment by GooseterV** [comment]
The license is actively licensing the content and has been for the past however-many months. The code is directly stolen, even if YOU have no come across a license or if YOU have not put a license in YOUR code. It is still in violation of MY license and a copy of OUR code therefore you are in the wrong here.

**Comment by monkxy** [comment]
> well its not on there so too bad

you're not going to have this attitude when your code gets taken down, look what happened to rx

**Comment by monkxy** [comment]
> The license is actively licensing the content and has been for the past however-many months. The code is directly stolen, even if YOU have no come across a license or if YOU have not put a license in YOUR code. It is still in violation of MY license and a copy of OUR code therefore you are in the wrong here.

exactly what goose is saying, it doesn't matter where you found it from the original code is still licensed meaning it is stolen regardless from where.

**Comment by GooseterV** [comment]
And regardless of whether or not the code that we have was licensed or not, if it was not licensed we have full copyright over it and can do as we see fit. If it was in fact licensed, which it was, here we are.

**Comment by ThatFruedDued** [comment]
The "Apollo Team" seems to be at a loss for how copyright laws work

**Comment by VillainsRule4000** [comment]
(@mamalona-lover is the bitch)
